clients/banking/src/components/CardListCells.tsx

```diff
@@ -153,7 +153,8 @@ export const FullNameAndCardTypeCell = ({ card }: { card: Card }) => {
                   {t("cards.format.virtualAndPhysical")}
                 </Tag>
 
-                {physicalCard?.statusInfo.status === "ToRenew" && (
+                {(physicalCard?.statusInfo.status === "ToRenew" ||
+                  physicalCard?.statusInfo.status === "Renewed") && (
```

Comment:
```tsx
match(physicalCard?.statusInfo.status)
  .with("ToRenew", "Renewed", () => <>…</>)
  .otherwise(() => null)
```

Is a bit more lisible when there are 2+ cases